### PR TITLE
fix: fix ui and some logic bugs

### PR DESCRIPTION
**Pull request by RanGojo**
# Summary

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

- [x] update full expanded ui styles
- [x] fix max button on 0 balance
- [x] add more priority to native tokens in sort logic
- [x] rounding route arrival time
- [x] fix some missing translations
- [x] avoid expanded mode when input <= 0.00

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.tsx [hunk 1]
```diff
@@ -168,7 +168,8 @@ export function FullExpandedQuote(props: PropTypes) {
               </Tooltip>
             </div>
             <StyledPriceImpact
-              size="xsmall"
+              size="small"
+              variant="full-expanded"
```
Comment (samobasquiat) [inline widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.tsx:172]: `PriceImpact` shouldn't be related to `variants` and such props. For example, you can pass just `outputColor`.

File: widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.Skeletons.tsx [hunk 2]
```diff
@@ -26,9 +26,7 @@ export function SkeletonHeader() {
         }}>
```
Comment (samobasquiat) [inline widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.Skeletons.tsx:26]: The design is always changing. I just checked it and the final version is like this: 
image

File: widget/embedded/src/store/slices/data.ts [hunk 3]
```diff
@@ -155,12 +155,22 @@ export const createDataSlice: StateCreator<
           }
         }
 
-        //3. sort by token popularity
+        const blockChainA = blockchainsMapByName.get(tokenA.blockchain);
+        const blockChainB = blockchainsMapByName.get(tokenB.blockchain);
```
Comment (samobasquiat) [inline widget/embedded/src/store/slices/data.ts:159]: I'm not familiar enough with sort tokens. It's better to check with @nikaaru

**Review by samobasquiat — CHANGES_REQUESTED** [review]

File: widget/embedded/src/store/slices/data.ts (hunk 3 above)
Comment (RanGojo) [reply widget/embedded/src/store/slices/data.ts:159]: done

File: widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.styles.ts [hunk 4]
```diff
@@ -207,8 +207,14 @@ export const Separator = styled('div', {
 });
```
Comment (samobasquiat) [inline widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.styles.ts:207]: It's better if the RightSection has a justify-content: flex-end property. The vertical line has a space with the horizontal line in the loading state.

Also, I'm not sure why `Steps` height is 102px. It seems to have changed in a previous commit, but it doesn't match figma.

**Review by samobasquiat — CHANGES_REQUESTED** [review]

**Review by samobasquiat — APPROVED** [review]

**Pull request merged**